Dockerfile

```diff
@@ -64,29 +64,29 @@ ENV LC_ALL en_US.UTF-8
 
 ENV HOME /data/olympia
 
-# version.json is overwritten by CircleCI (see circle.yml).
-# The pipeline v2 standard requires the existence of /app/version.json
-# inside the docker image, thus it's copied there.
-COPY version.json /app/version.json
 WORKDIR ${HOME}
 # give olympia access to the HOME directory
 RUN chown -R olympia:olympia ${HOME}
 
 # Set up directories and links that we'll need later, before switching to the
 # olympia user.
-RUN mkdir /deps \
-    && chown -R olympia:olympia /deps \
-    && rm -rf ${HOME}/src/olympia.egg-info \
-    && mkdir -p ${HOME}/src/olympia.egg-info \
-    && chown olympia:olympia ${HOME}/src/olympia.egg-info \
-    # For backwards-compatibility purposes, set up links to uwsgi. Note that
-    # the target doesn't exist yet at this point, but it will later.
-    && ln -s /deps/bin/uwsgi /usr/bin/uwsgi \
-    && ln -s /usr/bin/uwsgi /usr/sbin/uwsgi
+RUN <<EOF
```

Comment:
Just a cleaner way of writing the same code. + adding the symlink for package*.json